src/Courses/TopicsList.tsx

```diff
@@ -54,65 +56,119 @@ export const TopicsList: React.FC<TopicsListProps> = ({listOfTopics, flush, show
+                                </Row>
+                                {activeExtensions.length > 0 && (
+                                    <Row>
+                                        { userType !== UserRole.STUDENT ? (
+                                            <Link to={loc =>({pathname: `${loc.pathname}/settings`, selectedTopic: topic.id})}>
+                                                <Col>
+                                                    <p style={{color: 'black', fontStyle: 'italic'}}>
+                                                        This topic has {activeExtensions.length} active extension{activeExtensions.length > 1 && 's'}
+                                                    </p>
+                                                </Col>
+                                            </Link>
+                                        ) : (
+                                            <>
+                                                { _.find(activeExtensions, ['userId', userId]) !== undefined && (
+                                                    <Col>
+                                                        <p style={{color: 'black', fontStyle: 'italic'}}>
+                                                            You have an extension for this topic.
+                                                        </p>
+                                                    </Col>
+                                                )}
+                                            </>
+                                        )}
+                                    </Row>
+                                )}
+                            </Col>
                         </Row>
-                    </Col>
-                </>
-            ) : (
-                <>
-                    <Link to={loc =>({pathname: `${loc.pathname}/topic/${topic.id}`, state: {problems: topic.questions}})}>
-                        <h5>{topic.name}</h5>
-                    </Link>
-                    <MuiPickersUtilsProvider utils={MomentUtils}>
-                        <DateTimePicker
-                            style={{
-                                marginLeft: 'auto'
-                            }} 
-                            variant='inline'
-                            label='Start date'
-                            name={'start'}
-                            value={topic.startDate}
-                            onChange={()=>{}}
-                            onAccept={(date: MaterialUiPickersDate) => {
-                                if (!date) return;
-                                updateTopicField(topic, 'startDate', date.toDate());
-                            }}
-                            inputVariant='outlined'
-                            disabled={userType === UserRole.STUDENT}
-                        />
+                        <MuiPickersUtilsProvider utils={MomentUtils}>
+                            <DateTimePicker
```

Comment:
This is what I prefer over the controller above

Reply:
When I first encountered the issue I had noticed we did it differently in two different places